asmcomp/arm64/emit.mlp

```diff
@@ -683,6 +688,20 @@ let name_for_float_comparison = function
 
 (* Output the assembly code for an instruction *)
 
```

Comment:
This comment should remain attached to `emit_instr`, not become attached to `emit_stlr`.

If you wish to comment on `emit_stlr` (always a good idea), something like this should work:
```
(* Output a LR store of register [src] at address [base + addr].
   Since [stlr] does not support addressing modes, we compute [base + addr] explicitly. *)
```